lib/server.js

```diff
@@ -213,9 +213,9 @@ Server.prototype.setAllTasks = function (map) {
     var value = map[key]
     if (isObject(value)) {
       if (!isFunction(value.task)) {
-        throw new Error(
-          'ponos.Server.setAllTasks: No task function defined for ' + key
-        )
+        this.log.warn({
+          key: key
+        }, 'ponos.Server.setAllTasks: No task function defined')
       }
       this.setTask(key, value.task, value)
```

Comment:
`this.setTask` should not be called w/ an invalid `value.task`. The throw before was preventing this from being called.